src/lib/index.d.ts

```diff
@@ -25,6 +26,16 @@ export type WebsocketAdapterConfig = {
   }
 }
 
+export type HttpAdapterConfig = {
+  server: {
+    httpServer?: SyntaxError,
```

Comment:
```suggestion
    httpServer?: any,
```
why is it of type SyntaxError? Also you should update the [docs](https://github.com/asyncapi/glee/blob/master/docs/config-file.md)